### DISCUSSION TITLE
Neovim support

### DISCUSSION THREAD

**Discussion by joshmedeski** [withastro/roadmap#251] [Proposal]
I'd love to use Astro for an upcoming project, however, there is no neovim support yet.

Would anyone be interested in helping me set up the [vscode plugin](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode) to work with [coc.nvim](https://github.com/neoclide/coc.nvim)?

**Comment by delucis** [comment]
Hi @joshmedeski! 👋

We do list a couple of options [in our docs](https://docs.astro.build/en/editor-setup/#other-code-editors):

- [Vim Plugin](https://github.com/wuelnerdotexe/vim-astro) - Provides syntax highlighting, indentation, and code folding support for Astro inside of Vim or Neovim
- Neovim [LSP](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#astro) and [TreeSitter](https://github.com/virchau13/tree-sitter-astro) Plugins - Provides syntax highlighting, treesitter parsing, and code completion for Astro inside of Neovim

Do these help?

**Reply by joshmedeski** [reply]
Yes! I'll will give one of these options a good and give feedback where I can.
